airtime_mvc/application/models/Preference.php

```diff
@@ -498,7 +498,12 @@ public static function SetStationDescription($description)
 
     public static function GetStationDescription()
     {
-        return self::getValue("description");
+        $description = self::getValue("description");
+        if (!empty($description)) {
+            return $description;
+        } else {
+            return "Powered by Airtime Pro";
```

Comment:
https://core.trac.wordpress.org/ticket/9112 describes both context and comment lines in .po files. We would need to test if these show up in the Transifex interface.